### DISCUSSION TITLE
day1 gazebo 실행이 안됩니다.

### DISCUSSION THREAD

**Discussion by tukdy** [Road-Balance/gcamp_ros2_basic#28] [Q&A]
![gazebo error](https://user-images.githubusercontent.com/109137639/178441995-5d9ff93c-2ade-4bf5-9c0a-6a764fa063cd.png)

day1 학습했던 gazebo 입력시 화면이 뜨자마자 사라지며 D3D12: Removing Device라는 에러가 뜹니다.(pdf27쪽)
pdf29쪽에 나와있는 에러 해결 방법을 해봤는데도 되지않습니다.

**Comment by kimsooyoung** [comment]
사실 저도 처음보는 에러입니다. 

검색 결과 DirectX 관련 오류로 보이는데 혹 `$ rviz2`는 실행되는지, 에러가 난다면 해당 로그를 공유 부탁드립니다.

**Reply by tukdy** [reply]
$ rviz2는 정상작동 됩니다. gazebo 실행시 첫 주황색 화면이 나오고 바로 없어집니다.

**Reply by kimsooyoung** [reply]
`ps faux | grep gz` 를 통해 가제보 프로세스를 강제 종료한 뒤 다시 실행해보세요.
수업 시간 다시 한 번 다루겠습니다.

**Reply by kimsooyoung** [reply]
해결 되셨을까요?

**Reply by tukdy** [reply]
![가제보에러](https://user-images.githubusercontent.com/109137639/179147527-f2816fc4-2e16-45ec-ab2f-87cceaa3143f.png)
강제종료후 실행했지만 똑같은 결과입니다 ㅠ

**Reply by kimsooyoung** [reply]
windows 업데이트가 최신 상태이실까요??
